src/core/sock.c

```diff
@@ -292,6 +298,12 @@ static int nn_sock_setopt_inner (struct nn_sock *self, int level,
         return 0;
     }
 
+    if (level == NN_SOL_SOCKET && option == NN_SEC_ATTR) {
```

Comment:
Thinking about this, a transport option that can be used for UNIX IPC calls might be nice too.  Sort of a chmod or something.   But then we should think to make sure we can express this option portably.  I'm not sure how to do that.